frontend/pages/data-card/[dataCardId].tsx

```diff
@@ -56,9 +56,11 @@ export default function DataCard() {
         <PageWithTabs
           showCopyButton
           title={dataCard.name}
+          subheading={`ID: ${dataCard.id}`}
```

Comment:
Do we wan't the same with Access Requests and Schemas?

Reply:
They don't use this component in the same way - the Schemas page for example just uses the static title "Schemas", the reason we need this change for entries is because they have dynamic names and IDs